x-pack/plugins/maps/kibana.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "maps",
+  "version": "8.0.0",
+  "kibanaVersion": "kibana",
+  "configPath": ["xpack", "maps"],
+  "requiredPlugins": ["inspector"],
```

Comment:
what about `file_upload` (?)

ok. so we're not using it yet in the NP, but only in legacy